tideways/provision.sh

```diff
@@ -62,16 +62,16 @@ function restart_php() {
     for version in "7.0" "7.1" "7.2" "7.3" "7.4"
     do
         if [[ $(command -v php$version) ]]; then
-            service "php${version}-fpm" restart > /dev/null 2>&1
+            service "php${version}-fpm" restart
         fi
     done
     echo " * Restarting Nginx"
-    service nginx restart > /dev/null 2>&1
+    service nginx restart
 }
 
 function install_php_sqlite() {
     declare -a packages=()
-    for version in "7.0" "7.1" "7.2" "7.3" "7.4"; do
+    for version in "7.0" "7.1" "7.2" "7.3" "7.4" "8.0"; do
```

Comment:
Right now doesn't have support for php 8.0

Reply:
Tideways doesn't but this isn't installing Tideways for PHP 8.0, this is installing sqlite3, and a `php8.0-sqlite3` package exists.